src/main/java/bio/terra/cli/command/groups/AddUser.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cli.command.groups;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.utils.SamService;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra groups add-user" command. */
+@Command(name = "add-user", description = "Add a user to a group.")
+public class AddUser implements Callable<Integer> {
+  @CommandLine.Parameters(index = "0", description = "The email of the user.")
+  private String user;
+
+  @CommandLine.Option(names = "--group", required = true, description = "The name of the group")
```

Comment:
not this PR, just top of mind: no informed opinion, but we should write down some guidance about when we `--` prefix options, vs no prefix options, vs use command line parameters. Not urgent, but good for us to develop some consistency. (I care much more about have a consistent Terra CLI style that we can point to than what our actual guidelines are)
Sorry if this is already written and I've missed it.

Reply:
Agreed. Even more minor, but the varying capitalization of the parameter/option descriptions has also been bothering me.

It's not written down anywhere, yet. Feels like this should be low priority though -- work through functionality first, then we can talk about formatting later.